sdk/eventgrid/azure-eventgrid/azure/eventgrid/_operations/_patch.py

```diff
@@ -0,0 +1,179 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""Customize generated code here.
+Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
+"""
+import base64
+from typing import List, overload, Union, Any, Optional
+from ._operations import EventGridClientOperationsMixin as OperationsMixin
+from azure.core.messaging import CloudEvent
+from ..models._models import CloudEvent as InternalCloudEvent
+from ..models._patch import ReceiveResult, ReceiveDetails, PublishResult
+from azure.core.tracing.decorator import distributed_trace
+
+
+def _cloud_event_to_generated(cloud_event, **kwargs):
+    data_kwargs = {}
+
+    if isinstance(cloud_event.data, bytes):
+        data_kwargs["data_base64"] = base64.b64encode(cloud_event.data)  # base64 encode double check
+    else:
+        data_kwargs["data"] = cloud_event.data
+
+    internal_event = InternalCloudEvent(
+        id=cloud_event.id,
+        source=cloud_event.source,
+        type=cloud_event.type,
+        specversion=cloud_event.specversion,
+        time=cloud_event.time,
+        dataschema=cloud_event.dataschema,
+        datacontenttype=cloud_event.datacontenttype,
+        subject=cloud_event.subject,
+        **data_kwargs,
+        **kwargs
+    )
+    if cloud_event.extensions:
+        internal_event.update(cloud_event.extensions)
+    return internal_event
+
+
+class EventGridClientOperationsMixin(OperationsMixin):
+    @overload
+    def publish_cloud_events(
+        self,
+        topic_name: str,
+        body: List[CloudEvent],
+        *,
+        content_type: str = "application/cloudevents-batch+json; charset=utf-8",
+        **kwargs: Any
+    ) -> PublishResult:
```

Comment:
Wrote PublishResult model, did not generate from typespec b/c it is an empty model -- made a question for this on github. This will return None from the generated python code right now so print(publish_result)=None, but other languages return PublishResult (it is an empty model in typespec). Do I just leave publishResult here or have this operation return type None?